### PR TITLE
[BACK-2679] Resolve dependabot issues

### PR DESCRIPTION
**Pull request by darinkrauss**
- Resolve dependabot issues
- Update to node v20.11.0
- Add lint via jshint
- Resolve lint issues
- Add .travis.yml
- Add .nvmrc
- Bump module version
- https://tidepool.atlassian.net/browse/BACK-2679

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v20.11.0
```
Comment (jh-bate) [inline .nvmrc]: not `v20.9.0` ?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+dist: jammy
+
+language: node_js
+
+node_js:
+  - 20.10.0
```
Comment (jh-bate) [inline .travis.yml]: not `v20.9.0` ?

**Review by jh-bate — DISMISSED** [review]
just a note about the inconsistent node version from the stated `v20.9.0`

File: .nvmrc (hunk 1 above)
Comment (darinkrauss) [reply .nvmrc]: Using latest LTS, v20.11.0.

File: .travis.yml (hunk 2 above)
Comment (darinkrauss) [reply .travis.yml]: Using latest LTS, v20.11.0.

**Review by jh-bate — APPROVED** [review]
LGTM

**Pull request merged**